whc-shared/src/main/java/com/ibm/whc/deid/shared/pojo/config/masking/ZIPCodeMaskingProviderConfig.java

```diff
@@ -121,6 +122,30 @@ public void setMaskSuffixReplaceWithValidOnly(boolean maskSuffixReplaceWithValid
     this.maskSuffixReplaceWithValidOnly = maskSuffixReplaceWithValidOnly;
   }
 
+  @Override
+  public void validate() throws InvalidMaskingConfigurationException {
+    if (maskCountryCode == null) {
+      throw new InvalidMaskingConfigurationException(
+          "`maskCountryCode` must not be null");
+    }
+    if (maskReplaceWithNeighborNearestCount < 0.0) {

```

Comment:
wouldn't this be < 1 if 0 is not valid?  You can just say 0 or 1 since this is an int.

Reply:
done